static_src/actions/route_actions.js

```diff
@@ -108,5 +115,13 @@ export default {
       routeGuid,
       route
     });
+  },
+
+  error(routeGuid, err) {
```

Comment:
is this a general error for the routes API? trying to figure out why we need both this one and the one for the route create case.